docs/permissions.md

```
@@ -78,3 +78,40 @@ The [`tidy-review`](inputs-outputs.md#tidy-review), [`format-review`](inputs-out
     permissions:
       pull-requests: write
 ```
+
+## Auto-fix
+
+The [`auto-fix`](inputs-outputs.md#auto-fix) feature requires `contents: write` permission
+in addition to any other permissions needed for other features:
+
+```yaml
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)